imjoy/runners/subprocess.py

```diff
@@ -69,6 +68,11 @@ def setup_subprocess_runner(engine):
             os.makedirs(work_dir)
         plugin_env = os.environ.copy()
         plugin_env["WORK_DIR"] = work_dir
+        imjoy_path = str(HERE.resolve())
+        plugin_env["PYTHONPATH"] = (
+            imjoy_path + os.path.pathsep + plugin_env.get("PYTHONPATH", "")
```

Comment:
We should use `pathlib` consistently in all Python 3 compatible modules.

```suggestion
            str(imjoy_path / plugin_env.get("PYTHONPATH", ""))
```

Reply:
Not for here, since `os.path.pathsep` is not `os.path.sep`, we are concatenating the path with `:`, not `/`.

Reply:
Right. Sorry.

Reply:
No worry, if you know a better way to do that with pathlib, we can change later.